sdk/python/tests/integration/offline_store/test_offline_write.py

```diff
@@ -107,7 +109,8 @@ def test_writing_incorrect_schema_fails(environment, universal_data_sources):
 
 
 @pytest.mark.integration
-@pytest.mark.universal_online_stores
+@pytest.mark.universal_offline_stores(only=["file", "redshift"])
+@pytest.mark.universal_online_stores(only=["sqlite"])
```

Comment:
hm in this case you probably don't need to specify `universal_online_stores` then, since the default will just be `sqlite`?

Reply:
the integration tests default to redis